src/client/scripts/game/misc/timeutil.js

```diff
@@ -152,6 +152,49 @@ function isCurrentDateWithinRange(startMonth, startDay, endMonth, endDay) {
 		&& checkDate <= new Date(checkDate.getFullYear(), endMonth - 1, endDay);
 }
 
+function getTextContentFromTimeRemain(time) {
+	let seconds = Math.ceil(time / 1000);
+	let minutes = 0;
```

Comment:
I think the new methods added to timeutil.js could be moved to clock.js. timeutil.js I intended mainly for working with dates

Reply:
https://github.com/Infinite-Chess/infinitechess.org/pull/282/commits/8aa01c2c05a1b8af2597241946e6cbf1aaa3159
Clock data is now contained under `gamefile.clocks`

Reply:
I'd say keep it like this to reduce dependency hell, maybe move `getMinutesAndIncrementFromClock` to clock.js as it is only used by clock.js

Reply:
Perhaps we should then be reducing the dependancies of clock.js? `game` is only used for `game.getGamefile()` so we can solve that by passing the gamefile as a parameter. `onlinegame` is also not needed if we add a parameter.

But, if we want to hold off on resolving (some) of those dependancies for now, I still feel these new methods in `timeutil.js` don't fit right. Like, they only make sense with the context that they are working with gamefile clocks, whereas all the other methods in the script work with dates, UTC time, conversion from milliseconds to minutes and other generalized stuff. If you'd rather, do you want to make a new script `clockutil.js` with zero dependancies that stores the conversion methods related to gamefile clocks?

Reply:
I see what you mean. I've moved it into `clockutil.js` for now

Reply:
Thanks!